test/services/messengerSync/services/schoolSyncService.test.js

```diff
@@ -0,0 +1,107 @@
+const { expect } = require('chai');
+const commons = require('@schul-cloud/commons');
+
+const mockery = require('mockery');
+const app = require('../../../../src/app');
+const testObjects = require('../../helpers/testObjects')(app);
+const rabbitmqMock = require('../rabbitmqMock');
+
+//const schoolSyncService = app.service('schools/:schoolId/messengerSync');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected exception block, space or tab after '//' in comment.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5203703)